dev-python/busfactor/busfactor-0.2.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 eutils
+
+DESCRIPTION="A bus factor analyzer for Git repositories"
+HOMEPAGE="https://github.com/SOM-Research/busfactor"
+SRC_URI="https://timeraider4u.github.io/distfiles/files/${P}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE=""
+
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="dev-lang/python:2.7"
+RDEPEND="${DEPEND}
+	~dev-python/gitana-${PV}"
```

Comment:
no, it should be the same version for busfactor and gitana - otherwise it will not work properly.